src/main/java/ru/yandex/practicum/filmorate/controller/FilmController.java

```diff
@@ -52,4 +52,9 @@ public List<Film> getTopFilms(
             @Positive @RequestParam(value = "count", defaultValue = "10") Integer count) {
         return filmService.getTopFilms(count);
     }
+
+    @DeleteMapping("/{filmId}")
+    public void deleteFilmById(@PathVariable Integer filmId) {
```

Comment:
Предлагаю прописать имя переменной запроса
```suggestion
    public void deleteFilmById(@PathVariable("filmId}"}) Integer filmId) {
```